apps/daffio/src/app/docs/guide-docs/components/table-of-contents/table-of-contents.component.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  Input,
+} from '@angular/core';
+
+import { DaffioDoc } from '../../../shared/models/doc';
+
+@Component({
+  selector: 'daffio-guide-table-of-contents',
+  templateUrl: './table-of-contents.component.html',
+  styleUrls: ['table-of-contents.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class DaffioGuideTableOfContentsComponent {
+  /**
+   * The doc to render
+   */
+	@Input() doc: DaffioDoc;
```

Comment:
Can the input here just be the toc[]?

Reply:
It cannot. I need the `doc.id` to make the anchor tag links:
```
<a class="table-of-contents__item--level{{entry.lvl}}" routerLink="/{{doc.id}}" [fragment]="entry.slug">{{ entry.content }}</a>
```

Reply:
changed